src/test/java/redis/clients/jedis/tests/ShardedJedisPoolTest.java

```diff
@@ -54,8 +56,39 @@ public void checkConnections() {
     jedis.close();
     pool.destroy();
   }
-
+  
   @Test
+  public void checkConnectionsWithClientName() {
```

Comment:
Please format your code, you have a `make format` target in the makefile to do this. Make sure to commit only this files and now all the formatted ones.